src/get-dart/_mac.md

```diff
@@ -1,42 +1,60 @@
-[Install Homebrew,](https://brew.sh)
-and then run the following commands:
+
+### Install using Homebrew
+{: .no_toc}
+
+To install the Dart SDK, use [Homebrew][].
+
+Install Homebrew, then run the following command.
 
 ```terminal
-$ brew tap dart-lang/dart
-$ brew install dart
+$ brew install dart-sdk
```

Comment:
I haven't reviewed the rest of this PR, but note the existing/previous tap and install is what we recommend. The built-in `dart-sdk` recipe is not maintained by the Dart team and not kept up to date.

Reply:
According to the issue, the tap and install installs Dart 2.18.7. The `dart-sdk` install installed Dart 3.2.4.

Reply:
I'd like for the infra team to confirm this issue before we act on it. And as a general comment, I prefer PRs that are scoped to be more specific and don't bundle too many changes together.